src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -687,13 +693,131 @@ describe('ThreadView', () => {
+      renderComponent(discussionPostId);
+
+      const comment = await waitFor(() => screen.findByTestId('comment-comment-1'));
+      const sortWrapper = container.querySelector('.comments-sort');
+      const sortDropDown = within(sortWrapper)
+        .getByRole('button', { name: /Oldest first/i });
+
+      expect(comment)
+        .toBeInTheDocument();
+      expect(sortDropDown)
+        .toBeInTheDocument();
+    });
+
+    it('should not show sort dropdown if there is no response', async () => {
+      const commentId = 'comment-1';
+      renderComponent(discussionPostId);
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      axiosMock.onDelete(`${commentsApiUrl}${commentId}/`)
+        .reply(201);
+      await executeThunk(removeComment(commentId, discussionPostId), store.dispatch, store.getState);
+
+      expect(await waitFor(() => screen.findByText('No responses', { exact: true })))
+        .toBeInTheDocument();
+      expect(container.querySelector('.comments-sort'))
+        .not
+        .toBeInTheDocument();
+    });
+
+    it('should have only two options', async () => {
+      renderComponent(discussionPostId);
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
+
+      expect(dropdown)
+        .toBeInTheDocument();
+      expect(await within(dropdown)
+        .getAllByRole('button'))
+        .toHaveLength(2);
+    });
+
+    it('should be selected Oldest first and auto focus', async () => {
+      renderComponent(discussionPostId);
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
+
+      expect(dropdown)
+        .toBeInTheDocument();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Oldest first/i }))
+        .toBeInTheDocument();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Oldest first/i }))
+        .toHaveFocus();
+      expect(within(dropdown)
+        .getByRole('button', { name: /Newest first/i }))
+        .not
+        .toHaveFocus();
+    });
+
+    test('successfully handles sort state update', async () => {
+      renderComponent(discussionPostId);
+
+      expect(store.getState().comments.sortOrder)
+        .toBeFalsy();
+
+      await waitFor(() => screen.findByTestId('comment-comment-1'));
+      await act(async () => {
+        fireEvent.click(screen.getByRole('button', { name: /Oldest first/i }));
+      });
+      const dropdown = await waitFor(() => screen.findByTestId('comment-sort-dropdown-modal-popup'));
```

Comment:
These three lines are repeated in the above 3 test cases. Make a function and move these lines there.